docs/doc/21-load-data/01-s3.md

```diff
@@ -9,7 +9,7 @@ description:
 <img src="https://datafuse-1253727613.cos.ap-hongkong.myqcloud.com/load/load-data-from-s3.png" width="550"/>
 </p>
 
-Databend [COPY](../30-reference/30-sql/10-dml/dml-copy.md) command can read object files(CSV or Parquet file format) from Amazon S3 buckets.
+Databend [COPY](../30-reference/30-sql/10-dml/dml-copy-into-table.md) command can read object files(CSV or Parquet file format) from Amazon S3 buckets.
```

Comment:
We also support `json` file format, which can be added here by the way?

Reply:
Seems we don't have the json sample data?

```shell
> curl https://datafuse-1253727613.cos.ap-hongkong.myqcloud.com/data/books.parquet
<?xml version='1.0' encoding='utf-8' ?>
<Error>
        <Code>AccessDenied</Code>
        <Message>Access Denied.</Message>
        <Resource>datafuse-1253727613.cos.ap-hongkong.myqcloud.com/data/books.json</Resource>
        <RequestId>NjI5NDI4ZDhfZTFhNzM4MDlfYzJjY18yMjFkNDJl</RequestId>
        <TraceId>OGVmYzZiMmQzYjA2OWNhODk0NTRkMTBiOWVmMDAxODc0OWRkZjk0ZDM1NmI1M2E2MTRlY2MzZDhmNmI5MWI1OTVmYzc0YmEzZDg5YTVjMDA4YzA3ZWY5ZjQwZTE2ZGU1MTE5N2Q3ZmIwN2M0MWZlZWViZTBmZTIwMzc4M2U4NjU=</TraceId>
</Error>
```